### PR TITLE
Ignore .tox directory.  Used to be ignored by default w/ flake8 appar…

### PR DESCRIPTION
**Pull request by dannon**
…ently, but not anymore.

### CODE REVIEW

File: test/base/driver_util.py [hunk 1]
```diff
@@ -442,7 +442,7 @@ def setup_shed_tools_for_test(app, tmpdir, testing_migrated_tools, testing_insta
             tool_configs.remove(relative_migrated_tool_panel_config)
         for installed_tool_panel_config in INSTALLED_TOOL_PANEL_CONFIGS:
             tool_configs.append(installed_tool_panel_config)
-        from galaxy import tools  # noqa, delay import because this brings in so many modules for small tests
+        from galaxy import tools  # delay import because this brings in so many modules for small tests # noqa
```
Comment (nsoranzo) [inline test/base/driver_util.py]: Add `: E402` at the end?

Comment (dannon) [inline test/base/driver_util.py]: Sure, good to specify.  Going to look up the rest of them too.

Comment (nsoranzo) [inline test/base/driver_util.py]: Cool, was doing the same! My ack found:
lib/galaxy/model/mapping.py
lib/galaxy/webapps/galaxy/controllers/dataset.py
lib/galaxy/webapps/galaxy/controllers/forms.py
lib/galaxy/webapps/galaxy/controllers/library_common.py
scripts/db_shell.py

File: lib/galaxy/web/__init__.py [hunk 2]
```diff
@@ -28,3 +26,12 @@
...
 from framework.base import httpexceptions
+
+__all__ = ['url_for', 'error', 'expose', 'json', 'json_pretty',
+           'require_admin', 'require_login', 'expose_api', 'expose_api_anonymous',
+           'expose_api_raw', 'expose_api_raw_anonymous', '_future_expose_api',
+           '_future_expose_api_anonymous', '_future_expose_api_raw',
+           '_future_expose_api_raw_anonymous',
+           '_future_expose_api_anonymous_and_sessionless',
+           '_future_expose_api_raw_anonymous_and_sessionless', 'form',
+           'FormBuilder', 'httpexceptions']
```
Comment (nsoranzo) [inline lib/galaxy/web/__init__.py:37]: Awesome, thanks a lot!

**Pull request merged**